yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file should not be here.

Reply:
These files show up if you accidentally execute `yarn install` at the top level `kiali` directory.